types/lodash/util/uniqueId.d.ts

```diff
@@ -0,0 +1,25 @@
+declare namespace _ {
+    interface LoDashStatic {
+        /**
+         * Generates a unique ID. If prefix is provided the ID is appended to it.
+         *
+         * @param prefix The value to prefix the ID with.
+         * @return Returns the unique ID.
+         */
+        uniqueId(prefix?: string): string;
+    }
+
+    interface LoDashImplicitWrapper<TValue> {
+        /**
+         * @see _.uniqueId
```

Comment:
Maybe these comments are unnecessary now? We're *at* `_.uniqueId`.

Reply:
These comments were apparently made to write the description only once in the first overload of the method in the hope that IDEs will be smart enough to parse these `@see` annotations and inherit JSDoc somehow. See Microsoft/TypeScript/issues/407